sormas-backend/src/main/java/de/symeda/sormas/backend/region/RegionFacadeEjb.java

```diff
@@ -347,7 +348,7 @@ public void saveRegion(RegionDto dto, boolean allowMerge) throws ValidationRunti
 		Region region = regionService.getByUuid(dto.getUuid());
 
 		if (region == null) {
-			List<Region> duplicates = regionService.getByName(dto.getName(), true);
+			List<Region> duplicates = regionService.getByName(dto.getName(), countryService.getByReferenceDto(dto.getCountry()), true);
```

Comment:
This will result in a NullPointerException for regions that are not assigned to a country